content/SCALE/SCALEUIReference/SCALEDashboard.md

```diff
@@ -48,7 +45,7 @@ Use **Save** to retain any setting changes you make. Click on the **X** or on an
 Click on the <span class="iconify" data-icon="ic:baseline-assessment"></span> icon to display the report screen that corresponds to that widget. For example, clicking the <span class="material-icons-outlined">assessment</span> icon on the **CPU** widget opens the **Reports > CPU** screen. 
 
 ### System Information Widget
-The **System Information** widget displays general information on the SCALE system.   
+The **System Information** widget displays general information on the SCALE system, and includes a option to syncronize the system server time with TrueNAS SCALE time if they get out of sync.   
```

Comment:
```suggestion
The **System Information** widget displays general information on the SCALE system. It includes an option to synchronize the system server time with TrueNAS SCALE time if they get out of sync.   
```

Reply:
You know, I spelled it that way and darned if Hemingway didn't tell me I'd misspelled it!